src/test/regress/input/external_table.source

```diff
@@ -303,6 +303,13 @@ DROP OWNED BY test_role_issue_12748;
 DROP ROLE test_role_issue_12748;
 DROP PROTOCOL dummy_protocol_issue_12748;
 
+-- Test pg_dump's DROP EXTERNAL TABLE
+-- GitHub Issue #1025: https://github.com/arenadata/gpdb/issues/1025
+CREATE EXTERNAL TABLE issue_1025 (a int) location ('file://@hostname@@abs_srcdir@/data/no/such/place/badt1.tbl' ) format 'csv' (DELIMITER '|');
```

Comment:
Maybe better to set the schema name during table creation? As the schema name is used in subsequent command and in output file:
```sql
create external table public.issue_1025 (a int) ...
```